src/include/common/exception.h

```diff
@@ -128,76 +128,43 @@ class Exception : public std::runtime_error {
     }
   }
 
-  // Based on :: http://panthema.net/2008/0901-stacktrace-demangled/
-  static void PrintStackTrace(FILE *out = ::stderr,
-                              unsigned int max_frames = 63) {
-    ::fprintf(out, "Stack Trace:\n");
+  static void PrintStackTrace(FILE *out = ::stderr) {
```

Comment:
Can you provide a comment on what is this one is trying to do as you have deleted the previous reference link?

Besides, I know it is not you did this but it is actually bad to have the implementations in header files. Can you move them to the exception.cpp?